internal/graph/service.go

```diff
@@ -0,0 +1,67 @@
+package graph
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/syncromatics/kafmesh/internal/graph/generated"
+	"github.com/syncromatics/kafmesh/internal/graph/loaders"
+	"github.com/syncromatics/kafmesh/internal/graph/resolvers"
+
+	"github.com/99designs/gqlgen/graphql"
+	"github.com/99designs/gqlgen/graphql/handler"
+	"github.com/99designs/gqlgen/graphql/playground"
+	"github.com/go-chi/chi"
+	"github.com/pkg/errors"
+	"github.com/syncromatics/go-kit/log"
+)
+
+type Service struct {
+	port int
+	db   *sql.DB
+}
+
+func NewService(port int, db *sql.DB) *Service {
+	return &Service{port, db}
+}
+
+func (s *Service) Run(ctx context.Context) func() error {
+	router := chi.NewRouter()
```

Comment:
is chi cool? maybe we should determine a preferred router for golang apps. i think currently we use a bunch of different ones.

Reply:
It's just what the gqlgen used in their example. I haven't looked into to it too deeply.

Reply:
I'm guessing we could wire up any muxer here